documentation/docs/extensibility.md

```diff
@@ -1,86 +1,214 @@
 # Extensibility
 
-There are several possibilities for extensibility besides the **[very powerful configuration](configuration.md)**:
+When using one of the ready-made pipelines of project "Piper", you don't have to write custom pipeline code.
```

Comment:
Maybe we could add a summary in the form of a picture about the various options at the beginning.

Reply:
Good idea, I've added a proposal to the latest commit. Any suggestions about that?

Reply:
Looks pretty good. Not sure if we need the headings though